tests/get-acey-deucey.test.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
+
+const test = require("tape");
+const aceyDeuceyGameEngine = require("../");
+const getAceyDeucey = aceyDeuceyGameEngine.getAceyDeucey;
+
+test("get acey deucey", t => {
+    t.test("two dice", t => {
+        t.plan(4);
+        
+        const aceyDeucey = {isAceyDeucey: false};
```

Comment:
I generally like to name values like this with `expected` to make it clear that it's the expected output, not an input to the test function. How about `expectedAceyDeucey`? Your discretion.